### PR TITLE
Add inject to "bundle everything" list

### LINKED ISSUE #2872: Ensure inject.js (like parity.js) can operate stand-alone

### PR DESCRIPTION
**Pull request by jacogr**
Getting the following issues while trying to replicate (https://github.com/ethcore/parity/issues/2868 - at this point getting a diefferent issue, not the one listed, but it still doesn't work) -

```
external_"vendor_lib":1 Uncaught ReferenceError: vendor_lib is not defined(anonymous function) @ external_"vendor_lib":11 @ inject.js:1t @ inject.js:1(anonymous function) @ module.js_from_dll-reference_vendor_lib:1181 @ inject.js:5t @ inject.js:1(anonymous function) @ utf8.js:244138 @ inject.js:4t @ inject.js:1(anonymous function) @ utils.js:3935 @ inject.js:3t @ inject.js:1(anonymous function) @ requestmanager.js:281200 @ inject.js:13t @ inject.js:1(anonymous function) @ web3.js:281186 @ inject.js:11t @ inject.js:1(anonymous function) @ index.js:11175 @ inject.js:11t @ inject.js:1(anonymous function) @ inject.js:17(anonymous function) @ inject.js:35836 @ inject.js:10t @ inject.js:1(anonymous function) @ multi_inject:10 @ inject.js:1t @ inject.js:1n.0 @ inject.js:1(anonymous function) @ inject.js:1
194app.js:4 Uncaught TypeError: Cannot read property 'eth' of undefined
```

Obviously `inject.js` (injecting web3) like `parity.js` should not need the vendor stuff. (I have seen this internally as well.)

@ngotchac Could you take a look to make sure my approach here is right from a build prespective?

Fixes #2872

### CODE REVIEW

**Comment by ngotchac** [comment]
Ok so I removed the DLL used in dev in production builds, this should do the trick.
I also updated the `environment` index file so it uses the `PARITY_URL` if provided, so you can test that it works:

1. Run `PARITY_URL="127.0.0.1:3000" NODE_ENV="production" npm run build`
3. Run `echo "<script src='inject.js'></script><script>alert(window.web3 ? 'It works' : 'Something is wrong...')</script>" > .build/test.html`
2. Execute `node build-server`
3. Go to http://127.0.0.1:3000/test.html`

File: js/webpack.config.js [hunk 1]
```diff
@@ -41,7 +41,6 @@ module.exports = {
     'signaturereg': ['./dapps/signaturereg.js'],
     'tokenreg': ['./dapps/tokenreg.js'],
     // library
-    'inject': ['./inject.js'],
     'parity': ['./parity.js'],
```
Comment (jacogr) [inline js/webpack.config.js]: Remove parity.js as well?

File: js/webpack.config.js [hunk 2]
```diff
@@ -164,13 +164,13 @@ module.exports = {
...
         new webpack.optimize.CommonsChunkPlugin({
-          chunks: [ 'index' ],
+          chunks: ['index'],
           name: 'commons'
         })
       );
       plugins.push(
         new webpack.optimize.CommonsChunkPlugin({
-          chunks: [ 'parity' ],
+          chunks: ['parity'],
```
Comment (jacogr) [inline js/webpack.config.js]: As here - web3/inject & parity should be treated the same.

File: js/package.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{
+  {
```
Comment (jacogr) [inline js/package.json]: These 2 spaces look suspicious :)

File: js/webpack.config.js (hunk 2 above)
Comment (ngotchac) [inline js/webpack.config.js]: Ok that's true

**Pull request merged**